docs/source/architecture/journal.rst

```diff
@@ -2,476 +2,196 @@
 Journal
 *******
 
-The Journal is responsible for maintaining and extending the blockchain for the
-validator. This responsibility involves validating candidate blocks, evaluating
-valid blocks to determine if they are the correct chain head, and generating
-new blocks to extend the chain.
+The `journal` is the group of validator components that work together to handle
+batches and proposed blocks. These components are responsible for completing
+published blocks, publishing batches into blocks to extend the chain, and
+validating proposed blocks to determine if they are the correct chain head.
```

Comment:
Suggest changing "to determine if they are the correct chain head" to "to determine if they should be considered for the new chain head", since the consensus engine decides what the next chain head should be.